packages/release-please/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "release-please",
```

Comment:
That's the name of the actual `release-please` npm module, I think.  We should make this `release-please-bot` or something like that to avoid issues.